content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -7,36 +7,36 @@ alwaysopen = false
 
 {{%children style="h2" description="true"%}}
 
-This page provides download links to all of Cloudify's deployment packages, both Premium and Community.
+This page provides download links to all of {{< param product_name >}}'s deployment packages, both Premium and Community.
 
-Read about the differences between our [Cloudify versions](https://cloudify.co/download).
+Read about the differences between our [{{< param product_name >}} versions](https://cloudify.co/download).
 
 
 
 
-## Cloudify Premium (Premium download / Hosted service trial)
+## {{< param product_name >}} Premium (Premium download / Hosted service trial)
 
 Current Version: 5.1       [Release Notes](https://cloudify.co/release-notes-5-1-2020/)
 
 #### Downloads
 
-Cloudify **Premium Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-install-5.1.0ga.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-aio-docker-5.1.0ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-5.1.0ga.qcow2)
+{{< param product_name >}} **Premium Manager** Images:  [RPM](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-install-5.1.0ga.rpm)	|	[Docker](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-aio-docker-5.1.0ga.tar)	|	[QCOW](https://repository.cloudifysource.org/cloudify/5.1.0/ga-release/cloudify-manager-5.1.0ga.qcow2)
```

Comment:
Maybe parameter for premium version could be used here (`mgr_premium_title`).

Reply:
Consider adding more params in a follow-up review.